test_data/config.yaml

```diff
@@ -1,5 +1,5 @@
 credentials:
-    some_value_here: not_a_secret
+    some_value_here: not_secret
```

Comment:
Was this necessary b/c the entropy calculation with the new chars alerted on `not_a_secret`?

Reply:
Yeah, I don't think we need to be too concerned though because we now have the wordlist filtering.

Reply:
I'm more concerned that, we'll have large diffs in baseline's when people update detect-secrets. 

This isn't as concerning as changing a secret type like we did in #26, (where all old secrets were removed and re-added), but it is a little, especially if it reduces TP's to some extent. (We'll see what the data says though, I can't really say how it'll effect signal.)

Reply:
Why will we have large diffs? A lot of new secrets?

Reply:
If all the e.g. `not_a_secret` potential secrets disappear from existing baselines, then there is a possibility we will have large diffs, in the case of FP's that's great, in the case of TP's that would be a regression visible to users. (We can't really say there are minimal regressions without data though.)